src/adagrams.js

```diff
@@ -1,15 +1,107 @@
+// global variable
+const LETTER_POOL = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1
+}
+// the following line of code imports the library lodash
+// we use this library because we want to use the method sampleSize and the method cloneDeep 
+const _ = require('lodash');
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letterList = [];
+  for (let [letter, freq] of Object.entries(LETTER_POOL)) {
+    letterList.push(...Array(freq).fill(letter));
+  }
```

Comment:
Great work getting comfortable with JavaScript's syntax. It looks like you're getting use to looping over data and adding conditional logic ✅